### PR TITLE
New optimisation level 'Ov'

### PR DESCRIPTION
**Pull request by tj-sun**
which just does vectorisation and no loop motion

### CODE REVIEW

**Comment by miklos1** [comment]
If we are following the tradition of C compilers for naming optimisation levels, then I think we should follow it faithfully and use lowercase letters for the level "number", i.e. `Ov` instead of `OV`, similarly to `-Os` or `-Og` which are common to C compilers.

**Comment by miklos1** [comment]
Commit message has a typo and its first line is too long.

File: coffee/__init__.py [hunk 1]
```diff
@@ -174,6 +174,7 @@ def set_opt_level(optlevel):
 O1 = OptimizationLevel('O1', rewrite=1)
 O2 = OptimizationLevel('O2', rewrite=2, dead_ops_elimination=True)
 O3 = OptimizationLevel('O3', align_pad=True, **O2)
+OV = OptimizationLevel('Ov', align_pad=True)
```
Comment (miklos1) [inline coffee/__init__.py]: Another big `OV`

Comment (tj-sun) [inline coffee/__init__.py]: I'm surprised how this didn't give an error.. but amended.

Comment (miklos1) [inline coffee/__init__.py]: There is still a big `OV` in the diff...

**Comment by miklos1** [comment]
Can you squash these commits and fix the commit message?

Plus, you have at least one more big OV...

**Pull request merged**